docs/template_extension/CMakeLists.txt

```diff
@@ -32,4 +33,5 @@ if (ngraph_onnx_importer_FOUND)
     target_link_libraries(${TARGET_NAME} PRIVATE ${ONNX_IMPORTER_LIBRARIES})
     target_compile_definitions(${TARGET_NAME} PRIVATE NGRAPH_ONNX_IMPORT_ENABLED)
 endif()
+add_clang_format_target(${TARGET_NAME}_clang FOR_SOURCES ${template_extension_src})
```

Comment:
let's move beyond the `# [cmake:extension]`

Reply:
Fixed